bugs/b59870ca.txt

```diff
@@ -0,0 +1,368 @@
+# Verification Script
+
+https://github.com/canonical/cloud-init/commit/b59870ca3d8993612452a7b90687e2ed4ad0572a
+
+bug_b59870ca () {
```

Comment:
Nice reproducer test.
Can we somehow link  this document to 20200116/README.md probably by replacing the 
| SRU Manual #5 | [verification output](../bugs/sru-19.4.33-manual-5.txt) |

with 
| SRU Manual #5 | [verification output](../bugs/b59870ca.txt) |